summit.html

```diff
@@ -100,7 +100,9 @@ <h2>Day 1 (Wednesday, September 4)</h2>
                       real-world problems can Eiffel help out with?
                     </p>
                     <p>
-                      <b>Speaker:</b> TBD (Axis &amp; Ericsson)
+                      <b>Speakers:</b>
+                      Magnus Bäck (Axis Communications),
+                      Emil Bäckmark (Ericsson)
                     </p>
                   </div>
                 </details>
```

Comment:
A comment regarding the timing for this session. We think the first part of it will be ~40 minutes, and then 2x20 minutes about use cases in two companies. Should the break be postponed to at least 10.15? And then reconvene at 10.45? I suggest so.

Reply:
Yeah, I think that's a good idea. Updated accordingly.